### ISSUE TITLE
Switching to "High Contrast on" mode, the text "Make a Function" is missing

### ISSUE THREAD

**Issue by Helen09** [microsoft/pxt-ev3#465]
**Repro Steps:**
• Navigate to https://makecode.mindstorms.com/beta#editor
• Navigate to "More" and then click "High Contrast On"
• Click "Advanced" -> "Functions" and observe the new function box

**Expected result:**
The text "Make a Function" should exist
**Actual result:**
Switching to "High Contrast on" mode, the text "Make a Function" is missing
![4](https://user-images.githubusercontent.com/23466737/38489133-0365637a-3bd5-11e8-9485-6693d74983e1.gif)
